.github/workflows/android.yml

```diff
@@ -36,9 +36,9 @@ jobs:
         id: vars
         shell: bash
         run: |
-          echo "ANDROID_NDK_VERSION=22.1.7171670" >> $GITHUB_ENV
+          echo "ANDROID_NDK_VERSION=25.1.8937393" >> $GITHUB_ENV
```

Comment:
```suggestion
          ANDROID_NDK_VERSION=25.1.8937393
          echo "ANDROID_NDK_VERSION=${ANDROID_NDK_VERSION}" >> $GITHUB_ENV
          echo "ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk/${ANDROID_NDK_VERSION}" >> $GITHUB_ENV
```

Reply:
@nirvn did this get lost rebasing or was it broken?

Reply:
Lost when I pushed something locally and overwrote your change. But, it also turns out to not work for a reason I can't explain :/